test/e2e-cli/setup/teardown.ts

```diff
@@ -0,0 +1,4 @@
+module.exports = async function() {
+	// @ts-ignore
+	global.registryProcess.kill();
```

Comment:
I guess here the temporary (tempRoot ) can be unlinked as well ?

Reply:
I guess permission issue will be there to achieve it

Reply:
Since this is intended to run on CI, I think it's fine. Container will be discarded anyway

Reply:
But this for now an experiment, I'm sure there are a lot of room for improvement. We can PR changes in the future after see how it behaves after few weeks.

Reply:
> Since this is intended to run on CI, I think it's fine. Container will be discarded anyway

make sense 👍